dokuwiki-syntax/src/test/java/org/xwiki/contrib/dokuwiki/syntax/internal/parser/DokuWikiSpecificTest.java

```diff
@@ -29,7 +29,7 @@
  * @version $Id: 75488eb22c370f55a3e6bd9b38ec599be157b515 $
  */
 @RunWith(RenderingTestSuite.class)
-@RenderingTestSuite.Scope(value = "dokuwiki10.specific")//, pattern = "image1.test")
+@RenderingTestSuite.Scope(value = "dokuwiki10.specific", pattern = "list2.test")
```

Comment:
I think you should remove this. Its probably reducing the coverage.

Reply:
Alright. Yes, I missed undoing it.